biz.aQute.bndlib/src/aQute/bnd/build/Project.java

```diff
@@ -2729,13 +2800,15 @@ public Jar pack(String profile) throws Exception {
 	 */
 
 	public void baseline() throws Exception {
-		ProjectBuilder b = getBuilder(null);
-		for (Builder pb : b.getSubBuilders()) {
-			ProjectBuilder ppb = (ProjectBuilder) pb;
-			Jar build = ppb.build();
-			getInfo(ppb);
+		try (ProjectBuilder parentBuilder = getBuilder(null)) {
+			for (Builder b : parentBuilder.getSubBuilders()) {
+				ProjectBuilder pb = (ProjectBuilder) b;
```

Comment:
This reference is and cast are unnecessary. `Builder` has a `Jar build()` and `getInfo()` takes a `Reporter`